.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Test platform creation
+on: push
+jobs:
+  tests:
+    strategy:
+      matrix:
+        version: ["main", "stable"]
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install qibolab stable
+        if: ${{ matrix.version == 'stable' }}
+        run: |
+          python -m venv testenv
```

Comment:
Actually, why do you use a virtual environment?

I would have expected the self-hosted to run in some kind of container, or with some layer of external isolation. Isn't it the case?

Reply:
It seems to fail without the virtual environment

https://github.com/qiboteam/qibolab_platforms_qrc/actions/runs/6533280713/job/17738163307?pr=62